dev-tools/omdb/tests/test_all_output.rs

```diff
@@ -211,11 +211,16 @@ async fn test_omdb_success_cases(cptestctx: &ControlPlaneTestContext) {
             "cockroachdb_fingerprint",
             &initial_blueprint.cockroachdb_fingerprint,
         );
+
     let crdb_version =
         initial_blueprint.cockroachdb_setting_preserve_downgrade.to_string();
     if initial_blueprint.cockroachdb_setting_preserve_downgrade.is_set() {
         redactions.variable_length("cockroachdb_version", &crdb_version);
     }
+
+    let mut redactor = Redactor::new();
+    redactor.extra(redactions);
```

Comment:
I don't have the context for where else you'll be using the redactor in your future PRs but it feels like having the builder methods take `self` instead of `&mut self` would be more ergonomic, at least in this case.

Reply:
That seems quite reasonable.

Reply:
Ah so an issue is that we have places where we do conditional redactions, and setting `redactor = redactor.xyz()` feels a bit weird to me. In general whenever there are conditional things I tend to prefer `&mut self` builders. But 🤷🏽

Reply:
Yeah I feel like neither is ever the "right" answer. (The one unfortunate thing about the builder pattern.)